agent/nmap_agent.py

```diff
@@ -120,7 +120,9 @@ def _emit_network_scan_finding(self, scan_results, normal_results):
 
     def _emit_services(self, scan_results, domain_name):
         logger.info('sending results to %s', domain_name)
-        if scan_results is not None:
+        if scan_results is not None and\
```

Comment:
Use brackets to avoid the editor backslashs
```python
if (scan_results is not None and
                scan_results.get('nmaprun') is not None and
                scan_results['nmaprun'].get('host') is not None):
```